### PR TITLE
Dts remove property label

### PR DESCRIPTION
**Pull request by galak**
Rework the flash partition extraction so we can remove 'use-property-label'

### CODE REVIEW

File: scripts/dts/extract_dts_includes.py [hunk 1]
```diff
@@ -316,12 +316,6 @@ def extract_property(node_compat, yaml, node_address, prop, prop_val, names):
 
     node = reduced[node_address]
     label_override = None
```
Comment (erwango) [inline scripts/dts/extract_dts_includes.py]: You could clean up the 'label_override' here and lines 373/374 also I guess

File: scripts/dts/extract_dts_includes.py [hunk 2]
```diff
@@ -374,11 +368,7 @@ def extract_property(node_compat, yaml, node_address, prop, prop_val, names):
         def_label += '_' + label_override
 
     if prop == 'reg':
-        if 'partition@' in node_address:
-            # reg in partition is covered by flash handling
-            flash.extract(node_address, yaml, prop, def_label)
```
Comment (erwango) [inline scripts/dts/extract_dts_includes.py]: Not fond of this part of the patch. Would prefer to keep all extract calls inside extract function.

**Review by erwango — CHANGES_REQUESTED** [review]

File: scripts/dts/extract_dts_includes.py (hunk 2 above)
Comment (galak) [reply scripts/dts/extract_dts_includes.py]: The problem is we call extract per property in most cases, and special case flash handling.  We really should be handling flash per node, not per property.

File: scripts/dts/extract_dts_includes.py (hunk 1 above)
Comment (galak) [reply scripts/dts/extract_dts_includes.py]: Fixed

File: scripts/dts/extract_dts_includes.py (hunk 2 above)
Comment (galak) [reply scripts/dts/extract_dts_includes.py]: Add comments in about doing per node handling vs per property.

**Review by erwango — APPROVED** [review]

**Pull request merged**